modules/scip/9.2.0.bcr.1/patches/add_buildflags_h.patch

```diff
@@ -1,26 +1,33 @@
 diff --git a/src/scip/buildflags.h b/src/scip/buildflags.h
-index c686abcd20..c3fd91995a 100644
+index c686abcd20..1ea6a4fbf4 100644
 --- a/src/scip/buildflags.h
 +++ b/src/scip/buildflags.h
-@@ -29,9 +29,36 @@
+@@ -29,10 +29,44 @@
  
  /*---+----1----+----2----+----3----+----4----+----5----+----6----+----7----+----8----+----9----+----0----+----1----+----2*/
  
 +
 +#if defined(__GNUC__) && defined(__linux__)
 +#define SCIP_OSTYPE "Linux"
++#define SCIP_COMP "GCC"
```

Comment:
Nit: What about Linux+Clang (which is for example the standard at Google)?

Reply:
Done